subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_connmon.c

```diff
@@ -14,89 +15,66 @@
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(lwm2m_connmon, CONFIG_LWM2M_CLIENT_UTILS_LOG_LEVEL);
+#define IP_ADDR_LENGTH 46
+#define APN_LENGTH 64
+#define FW_VERSION_LENGTH 50
 
 static struct modem_param_info modem_param;
 static struct k_work modem_data_work;
 static struct k_work modem_signal_work;
 static int32_t modem_rsrp;
 
+static char *ip_addr[IP_ADDR_LENGTH];
+static char *apn[APN_LENGTH];
+static char *fw_version[FW_VERSION_LENGTH];
+
 /* LTE-FDD bearer & NB-IoT bearer */
 #define LTE_FDD_BEARER 6U
 #define NB_IOT_BEARER 7U
 
 static uint8_t bearers[2] = { LTE_FDD_BEARER, NB_IOT_BEARER };
 
+static void connmon_data_init(void)
+{
+	lwm2m_engine_create_res_inst("4/0/1/0");
```

Comment:
This is probably done in same way in other places as well, but it would be nicer to have constant strings for the paths and use those instead of having the same path string in multiple places. At least to me for example "4/0/7/0" means nothing and it would be easy to have an error somewhere (use incorrect path).